internal/jennies/java/jennies.go

```diff
@@ -0,0 +1,59 @@
+package java
+
+import (
+	"github.com/grafana/codejen"
+	"github.com/grafana/cog/internal/ast/compiler"
+	"github.com/grafana/cog/internal/jennies/common"
+	"github.com/spf13/cobra"
+)
+
+const LanguageRef = "java"
+
+type Config struct {
+	Debug bool
+
+	GenGettersAndSetters bool
+}
+
+type Language struct {
+	config Config
+}
+
+func (config Config) MergeWithGlobal(global common.Config) Config {
+	newConfig := config
+	newConfig.Debug = global.Debug
+
+	return newConfig
+}
+
+func New() *Language {
+	return &Language{config: Config{}}
+}
+
+func (language *Language) RegisterCliFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&language.config.GenGettersAndSetters, "getters-and-setters", true, "Generate getters and setters for types")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&language.config.GenGettersAndSetters, "java-getters-and-setters", true, "Generate getters and setters for types")
```

Just to make it obvious that this flag is only used for the Java output